### PR TITLE
feat(template): added pf1 template

### PR DESCRIPTION
**Pull request by LeCuay**
Missing a few sanity checks and community feedback but so far seems good

> Notes for feedback: Check always last image (scroll down) to see the final look

image

### CODE REVIEW

**Comment by EddieDover** [comment]
Don't forget to update TEMPLATE_LIST.md

**Comment by dmrickey** [comment]
Since the mod doesn't support `rowspan`, if you can manually split the vision into multiple columns that will help with cases like the first row of `Borug`. It might make the result look a bit weird if there are value(s) in row 1/3 but not row 2, but I think overall that will give a better result for trying to view the data in the rest of the columns so they're not "too tall" for an actor that has a lot of various vision abilities.

**Comment by dmrickey** [comment]
Other suggestion from discord ~~Vision~~ -> `Senses`

File: example_templates/pf1/pf1-template.json [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+        "name": "Name",
+        "type": "direct",
+        "header": "show",
+        "text": "name {newline} {i} race.name {/i}"
+      },
+      {
+        "name": "STR",
+        "type": "direct",
+        "header": "show",
+        "text": "system.abilities.str.value"
```
Comment (bhudnell) [inline example_templates/pf1/pf1-template.json:24]:
```suggestion
        "text": "system.abilities.str.total"
```

Comment (bhudnell) [reply example_templates/pf1/pf1-template.json:24]: Same for the other 5 abilities

**Comment by EddieDover** [comment]
> Since the mod doesn't support `rowspan`, if you can manually split the vision into multiple columns that will help with cases like the first row of `Borug`. It might make the result look a bit weird if there are value(s) in row 1/3 but not row 2, but I think overall that will give a better result for trying to view the data in the rest of the columns so they're not "too tall" for an actor that has a lot of various vision abilities.

Or wait for #39

**Comment by EddieDover** [comment]
Also @LeCuay @dmrickey feel free to add me on discord if you want. My handle is the same as my github name (if you're not already in my server).

**Comment by LeCuay** [comment]
Okay this is how it looks on [d1f6e9a](https://github.com/EddieDover/fvtt-party-sheet/pull/36/commits/d1f6e9a8e0f99bba70e64c1f52edac4fa79b646a)

image

File: example_templates/pf1/pf1-template.json (hunk 1 above)
Comment (LeCuay) [reply example_templates/pf1/pf1-template.json]: Done, thanks for the feedback!

**Comment by LeCuay** [comment]
Added few icons
image

**Review by EddieDover — DISMISSED** [review]

**Comment by LeCuay** [comment]
This is using Pro FontAwesome icons, I need to revisit

**Comment by LeCuay** [comment]
> ⚠️ This is using Pro FontAwesome icons, I need to revisit

Okay now is ready

**Review by EddieDover — APPROVED** [review]

**Pull request merged**